.github/workflows/cicd-prod.yml

```
@@ -1,6 +1,8 @@
-name: MovieBookie CD Pipeline
+name: MovieBookie CI/CD For Prod Pipeline
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)